builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -1133,12 +1143,17 @@ def prepareServerlessYml(aConfig, env, configLoader, envDeploymenDescriptor) {
   }
 
     def doc = deploymentDescriptor  ? readYaml(text: deploymentDescriptor ) : [:] // If no descriptor present then simply making an empty one. The readYaml default behavior is to return empty string back that is harful as Map not String is expected below
+    def logStreamer = configLoader.JAZZ.PLATFORM.AWS.KINESIS_LOGS_STREAM.PROD
+
+    def destLogStreamArn = configLoader.AWS.ACCOUNTS.find{ it.ACCOUNTID == aConfig.accountId}.PRIMARY ? logStreamer : 
+                            accountDetails.REGIONS.find{it.REGION == aConfig.region}.LOGS.PROD
+ 
 
     context =["environment_logical_id": env,
             "INSTANCE_PREFIX": configLoader.INSTANCE_PREFIX,
             "REGION": aConfig.region,
             "cloudProvider": "aws",
-            "kinesisStreamArn": configLoader.JAZZ.PLATFORM.AWS.KINESIS_LOGS_STREAM.PROD,
+            "kinesisStreamArn": destLogStreamArn,
```

Comment:
Since all accounts including primary have this setup. We can use: `accountDetails.REGIONS.find{it.REGION == aConfig.region}.LOGS.PROD` and not need to define logStreamer and destLogStreamArn above.